tracing-futures/src/lib.rs

```diff
@@ -381,6 +381,18 @@ impl<T> Instrumented<T> {
         &mut self.inner
     }
 
+    /// Get a pinned reference to the wrapped type.
+    #[cfg(feature = "std-future")]
+    pub fn inner_pin(self: Pin<&Self>) -> Pin<&T> {
```

Comment:
Take it or leave it: regarding naming, I'm on the fence about whether it's better for these to be consistent with the type's other methods (`inner_pin`/`inner_pin_mut`), or for them to be consistent with `pin-project` (maybe `project_inner_ref()`/`project_inner()`?)...what do you think?

Reply:
I didn't quite know what to call them, so came up with this. It's completely your call what you prefer. I don't know if the word `project` has an as easily identifiable meaning to everyone as `pin`  though. In the sense that I think people might have to check the documentation a bit more often to figure out what this does, but the pin API is still young, so maybe it will be a common vocabulary one day and for that adoption is the first step.

Reply:
tokio and futures use the name [`get_pin_mut`](https://github.com/tokio-rs/tokio/blob/master/tokio/src/io/util/buf_reader.rs#L81) for those methods, so I think `get_pin_ref`/`get_pin_mut` is preferable.

Reply:
Since the other methods are `inner` and `inner_mut`, we should probably name these `inner_pin_ref` and `inner_pin_mut` for consistency, then?